sympy/integrals/integrals.py

```diff
@@ -1174,7 +1174,7 @@ def _eval_lseries(self, x, logx=None, cdir=0):
             yield integrate(term, *expr.limits)
 
     def _eval_nseries(self, x, n, logx=None, cdir=0):
-        expr = self.as_dummy()
+        expr = self
```

Comment:
If we are not updating `expr` anywhere, it would make sense to not define a new variable and just use `self` in place of `expr` ?